### ISSUE TITLE
Disable mini.indentscope

### ISSUE THREAD

**Issue by pbnj-dragon** [LunarVim/bigfile.nvim#21]
[mini.indentscope](https://github.com/echasnovski/mini.indentscope) is a plugin that shows the current indentation scope (i.e. it highlights the current indent guide).

This plugin is enabled by default in LazyVim and it seems to cause performance issues on large files, from my experience.

**Comment by LostInTheLogs** [comment]
You can add a custom feature in the config by adding a table to the features array in setup(), it's in the readme

something like this should work
```lua
{
name = "mini.indentscope",
disable = function (buf)
  vim.b[buf].miniindentscope_disable = true
end
}
```

If more users come to us with the same request, we'll add it to the builtin list

**Comment by krishna-bala** [comment]
I also am using LazyVim and wanted to add a custom feature to disable `mini.indentscope`. But I couldn't figure out a way to conveniently append to the default `features`. I ended up copying the entire default features and adding this extra custom feature, like so:
```lua
return {
  "LunarVim/bigfile.nvim",
  opts = {
    filesize = 2,
    pattern = { "*" },
    features = {
      "indent_blankline",
      "illuminate",
      "lsp",
      "treesitter",
      "syntax",
      "matchparen",
      "vimopts",
      "filetype",
      {
        name = "mini_indentscope",
        disable = function(buf)
          vim.b[buf].miniindentscope_disable = true
        end,
      },
    },
  },
}
```

I tried a few other methods, but since the `setup` function uses `local config = vim.tbl_deep_extend("force", default_config, overrides or {})`, if I specify `features` in `opts` it will overwrite the `default_config.features` completely.

tl;dr: it would be nice to be able to append custom features on to the default feature list, rather than have to recreate the entire feature list. If I figure out how to do that, I'll send out a PR for discussion.
